README.md

```diff
@@ -170,6 +178,22 @@ Do not hesitate to contact
 need help with the integration of your PDE solver.
 
 
+External Matrix Equation Solvers
+--------------------------------
+
+pyMOR also provides bindings to matrix equation solvers (in `pymor.bindings`),
+which are needed for the system-theoretic methods and need to be installed
+separately. The following solver libraries are included:
```

Comment:
I would also prefer to make it more clear that we only include bindings and not the libraries themselves libraries. So maybe `Bindings for the following solver libraries are included"?

Reply:
IMHO the double dash makes it more clear to be range as opposed to 2013 minus 2019.

Reply:
Sorry, this was supposed to go one up.

Reply:
So, the correct mark would be an 'en dash' which usually has about the width of the letter 'N'. My impression is that '--' makes it stand out too much. I tried to find some official recommendations how to use dashes on mechanical typewriters, but could not find anything on the internet. In any case, I would say that using some unicode dash character is out of the question ..

Reply:
OK I will take it out.